t/fork.t

```diff
@@ -12,7 +12,7 @@ BEGIN {
      $Config::Config{useithreads} and
      $Config::Config{ccflags} =~ /-DPERL_IMPLICIT_SYS/
     );
-  if ( $can_fork ) {
+  if ( $can_fork && !$Devel::Cover::VERSION ) {
```

Comment:
Could you elaborate in the commit message why we would want to skip this when doing code coverage?

Reply:
At least on 'MSWin32', this happens:

```
C:\>set harness
HARNESS_OPTIONS=c:j
HARNESS_PERL_SWITCHES=-MJSON::PP -MDevel::Cover
HARNESS_TIMER=1

C:\>prove -bl t\fork.t
[19:45:17] t\fork.t .. 1/8
Unfortunately, Devel::Cover does not yet work with threads.  I have done
some work in this area, but there is still more to be done.

[19:45:17] t\fork.t .. Dubious, test returned 3 (wstat 768, 0x300)
Failed 7/8 subtests
[19:45:18]

Test Summary Report
-------------------
t\fork.t (Wstat: 768 Tests: 1 Failed: 0)
  Non-zero exit status: 3
  Parse errors: Bad plan.  You planned 8 tests but ran 1.
Files=1, Tests=1,  2 wallclock secs ( 0.05 usr +  0.03 sys =  0.08 CPU)
Result: FAIL
```
In looking at this now, I'm not sure if it's only 'MSWin32', but I can gate it to just that (via `$^O`) and note the error message in the commit.